packages/manager/apps/hub/webpack.config.js

```diff
@@ -70,6 +70,11 @@ module.exports = (env = {}) => {
       mainFields: ['module', 'browser', 'main'],
     },
     plugins: [
+      new webpack.ContextReplacementPlugin(
+        /moment[/\\]locale$/,
+        /cs|de|en-gb|es|es-us|fi|fr-ca|fr|it|lt|pl|pt/,
```

Comment:
Based on the following list: https://github.com/ovh/manager/blob/3297580738182ce4284efa189840c30c9d02041e/packages/manager/tools/component-rollup-config/src/plugins/translation-utils.ts#L7-L20

Reply:
Could this be declared in the common webpack config ? so it can be reused in all managers

Reply:
Since our webpack configuration is already glued with the Rollup configuration I think I will be able to get access to this list but I think it's not a good solution.

So as you've suggested I will integrate this list directly into the webpack configuration.
Thanks for the feedback.

Reply:
After digging around it I didn't any option to factorize this code snippet.

Let me know if you have some ideas to improve it.

For now I will distribute this enhancement across other webpack configuration.

Thanks!